src/main/java/com/entropyteam/entropay/employees/services/VacationService.java

```diff
@@ -81,7 +89,9 @@ public void discountVacationDebit(Employee employee, Integer totalDays) {
                     vacation.setDeleted(true);
                     totalDays -= debit;
                 }
-                vacationRepository.save(vacation);
+                Vacation savedEntity = vacationRepository.save(vacation);
+                LOGGER.info("Vacation debit discounted, employeeId: {}, year: {}, : available days before discount: {}, available days after discount: {}",
+                        savedEntity.getEmployee().getId(), savedEntity.getYear(), availableDays, vacationRepository.getAvailableDays(employee.getId()));
```

Comment:
Same as before, remove the `vacationRepository.getAvailableDays(employee.getId())` from the for iteration